src/applications/letters/components/DownloadLetterLink.jsx

```diff
@@ -69,31 +110,61 @@ export class DownloadLetterLink extends React.Component {
     }
 
     return (
-      <div>
-        <div className="form-expanding-group form-expanding-group-open">
-          <TransitionGroup>
-            {message ? (
-              <CSSTransition
-                classNames="form-expanding-group-inner"
-                appear
-                timeout={{
-                  appear: 700,
-                  enter: 700,
-                }}
-                exit={false}
-              >
-                {message}
-              </CSSTransition>
-            ) : null}
-          </TransitionGroup>
-        </div>
-        <VaButton
-          className="vads-u-margin-top--1 vads-u-margin-bottom--3"
-          disabled={buttonDisabled}
-          text={buttonText}
-          onClick={this.downloadLetter}
-        />
-      </div>
+      <Toggler.Hoc toggleName={Toggler.TOGGLE_NAMES.lettersPageNewDesign}>
+        {toggleValue =>
+          toggleValue ? (
+            <div className="form-expanding-group form-expanding-group-open">
+              <TransitionGroup>
+                {message ? (
+                  <CSSTransition
+                    classNames="form-expanding-group-inner"
+                    appear
+                    timeout={{
+                      appear: 700,
+                      enter: 700,
+                    }}
+                    exit={false}
+                  >
+                    {message}
+                  </CSSTransition>
+                ) : null}
+              </TransitionGroup>
+              <VaButton
+                className="vads-u-margin-y--0"
+                disabled={buttonDisabled}
+                text={buttonText}
+                onClick={this.downloadLetter}
+              />
+            </div>
+          ) : (
+            <div>
+              <div className="form-expanding-group form-expanding-group-open">
+                <TransitionGroup>
+                  {message ? (
+                    <CSSTransition
+                      classNames="form-expanding-group-inner"
+                      appear
+                      timeout={{
+                        appear: 700,
+                        enter: 700,
+                      }}
+                      exit={false}
+                    >
+                      {message}
+                    </CSSTransition>
+                  ) : null}
+                </TransitionGroup>
+              </div>
+              <VaButton
+                className="vads-u-margin-top--1 vads-u-margin-bottom--3"
+                disabled={buttonDisabled}
+                text={buttonText}
+                onClick={this.downloadLetter}
+              />
```

Comment:
@sh0wn and I rallied around the idea of leaving the Benefit Summary and Service Verification Letter a button download while developing the feature. This was a departure from the Figma files but we leaned into it for two reasons:

1. It was a cleaner UX. Users only had to click once to download their letter.
2. This letter is an outlier, where users are effectively submitting a form. Broadly speaking, forms are submitted by pressing a button, then something happens.

This workflow was presented to the VA accessibility specialists at the May 22nd sync and got agreement from peers and VA's accessibility lead.